dvc/remote/azure.py

```diff
@@ -16,23 +15,36 @@
 class AzureRemoteTree(BaseRemoteTree):
     scheme = Schemes.AZURE
     PATH_CLS = CloudURLInfo
-    REQUIRES = {"azure-storage-blob": "azure.storage.blob"}
+    REQUIRES = {
+        "azure-storage-blob": "azure.storage.blob",
+        "azure-cli-core": "azure.cli.core",
```

Comment:
Worried about this dependency being too heavy, might go back to plain knack solution instead.